modules/tinymce/src/core/main/ts/commands/LineHeight.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Fun, Optional } from '@ephox/katamari';
+import { Compare, Css, PredicateFind, SugarElement, SugarNode } from '@ephox/sugar';
+import Editor from 'tinymce/core/api/Editor';
+import * as CaretFinder from '../caret/CaretFinder';
+import * as NodeType from '../dom/NodeType';
+
+const findFirstCaretElement = (editor: Editor): Optional<Node> => CaretFinder.firstPositionIn(editor.getBody()).map((caret) => {
```

Comment:
I was under the understanding that the code in `FontCommands` was about to be torn up / replaced as part of #5983 or the work we do to replace #5983. Rather than abstracting out some code that's (hopefully) about to be deleted, I left comments on both Jiras saying "hey when the real solution is implemented, abstract it out and port the fixes here too"

Reply:
No, there are no plans to do that for now. Eventually, but it'll probably be a few minor versions away. Also just because something is planned to be destroyed, isn't a very good reason to create duplicate code (different story if it's in progress, but then this should have been blocked until it was completed).

Reply:
This also reminds me, @ltrouton we need to close the above PR since it's blocked and back on the drawing board.